cognee/tasks/translation/translate_content.py

```
@@ -0,0 +1,657 @@
+    def __init__(self):
+        if self._client is None:
+            try:
+                from azure.ai.translation.text import TextTranslationClient, TranslatorCredential
+                from azure.core.credentials import AzureKeyCredential
+
+                key = os.getenv("AZURE_TRANSLATE_KEY")
+                endpoint = os.getenv("AZURE_TRANSLATE_ENDPOINT")  # optional for global
+                region = os.getenv("AZURE_TRANSLATE_REGION")      # optional; required for some resources
+
+                if not key:
+                    raise AzureConfigError("Azure Translate key (AZURE_TRANSLATE_KEY) is required.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)